activity-6/app.js

```diff
@@ -0,0 +1,11 @@
+// p.86 - 6. changing element attributes
+
+// store an image element
+
+const image = document.getElementById("image");
+const input = document.getElementById("input");
+const button = document.getElementById("submit");
+
+submit.addEventListener("click", () => {
```

Comment:
You use the variable name _button_ but your event listener is attached to _submit_. Have you run this code?

Reply:
thanks - not sure what the correct procedure is to correct but I did it in vs code, committed with note and pushed